.env-devel

```diff
@@ -92,6 +92,7 @@ PAYMENTS_AUTORECHARGE_DEFAULT_MONTHLY_LIMIT=10000
 PAYMENTS_AUTORECHARGE_DEFAULT_TOP_UP_AMOUNT=100.0
 PAYMENTS_AUTORECHARGE_ENABLED=1
 PAYMENTS_AUTORECHARGE_MIN_BALANCE_IN_CREDITS=100
+PAYMENTS_FINANCE_DEPARTMENT_EMAIL=none
```

Comment:
THOUGHT: 
here we want to nullify an environ variable in a `.env` (dot-env). Unfortnunately there seems to be no standard to do that .  Note that this will transform to `None` because you are loading from pydantic. There are many other possibilities e.g. `null` will also work or `None` ...